src/containers/__Routing/Routing.js

```diff
@@ -55,6 +55,7 @@ const RoutingBody = (props) => {
         func: () => {
             loggingContext.addLog("Voice Command: Logging out")
             appContext.mongoHook.logout()
+            history.push("/logged out with voice")
```

Comment:
`history.push(url)` is for navigating to different pages using a url path. For the logout command we don't need to navigate to a URL since our Router will do that for us, and I believe that "/logged out with voice" is an invalid URL path. I'm not entirely sure but I don't think that URLs can contain spaces. If you wanted a specific logout page you could change this to "/logged_out_with_voice" and create a route for it

Reply:
yeah, just change it to history.push("/");